pypeclub/OpenPype
# Slate Generator

# List of tasks
- [x] reviewing old code
- [x] creating example slate from clients brief
- [x] scalable template
- [x] develop plugin starting after burnins
- [x] adding settings 

# testing notes
1. open Settings/project/global/Publish plugins/Extract Review
2. open Output definitions and go to h264
3. add `Add generated slate frame` to Tags attribute
4. Open Standalone publisher and select context on task > chose `render` family
5. add any files to publish
6. switch off Validate Frame Lenght if you need and publish
7. look to published files and see mp4 file wwith `*h264MainSlate`

# Future features
- generate slate on exr with any colorspace
- settings with `Generate preview` button on slate profile

closes https://github.com/pypeclub/OpenPype/issues/1832

[cuID:yjx9f8]
